SEED-platform/seed
# Speed up inventory beta page

#### Any background context you want to provide?

DCs inventory beta page is slow ([issue](https://github.com/SEED-platform/seed/issues/3000#issuecomment-1104192349)). There's a lot that can be speed up about this page. This PR tackles the lower hanging fruit, and drop the load time from 15s to 6s on my local version with the DC db.

#### What's this PR do?

1. (45b7ffd) create and use a "brief" version of all the column_list_profiles and only get the full version of the one we need.
2. (1253a4a  to f3b3637) Only get the properties once instead of twice.
3. (6b556cc) Remove unused `GET  org`
4. (5638852) create and use a "brief" version of get org

#### How should this be manually tested?
To  get the speed up, your going to need to get DC db on your local machine and compare this branches load time to master.
Else, open the network tab of devtools and notice the lack of redundant calls. 

#### What are the relevant tickets?

[Slow times first doumented](https://github.com/SEED-platform/seed/issues/3000#issuecomment-1104192349)
[Investigation began](https://github.com/SEED-platform/seed/issues/3253)

#### Screenshots (if appropriate)
